lib/Runtime/Base/ThreadContext.h

```diff
@@ -423,7 +423,24 @@ class ThreadContext sealed :
 #endif
 #endif
 
+public:
+    Js::PropertyRecord const * GetEmptyStringPropertyRecord()
+    {
+        if (!emptyStringPropertyRecord)
+        {
+            emptyStringPropertyRecord = this->UncheckedAddPropertyId(_u(""), 0);
```

Comment:
Is this guaranteed to happen prior to any user code? Or could I write some JS which immediately does something like `global[''] =1` to force the empty string to get a property record, and then this line here would attempt to create an additional one?

Reply:
Yes, this is possible. Updated.